test/e2e/runner/docker.go

```diff
@@ -0,0 +1,219 @@
+// nolint: gosec
```

Comment:
It's not clear to me what this is suppressing. Can we put it closer to the specific use, or is it something very general? (Guessing maybe something to do with the template?)

Reply:
I've moved all of these to their specific uses now.